types/audit-details.d.ts

```diff
@@ -68,15 +68,15 @@ declare global {
           wastedMs?: number;
           wastedBytes?: number;
         };
-        diagnostic?: Diagnostic;
+        debug?: DebugData;
       }
 
       /**
        * A details type that is not rendered in the final report; usually used
-       * for including diagnostic information in the LHR. Can contain anything.
+       * for including debug information in the LHR. Can contain anything.
        */
-      export interface Diagnostic {
-        type: 'diagnostic';
+      export interface DebugData {
+        type: 'debug';
```

Comment:
if we go with type `DebugData` maybe this string should be `debugdata` and the property name `debugData`?

It's a little confusing having both `debug` and `debugData` as reasonable local variable names

Reply:
agreed, `debugdata` SGTM :)